Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang:1.11.5 AS builder
-ADD . /src
-RUN cd /src \
- && go test . \
- && go build -o binary
+FROM golang:alpine as builder
+RUN apk add git && \
+	go get -d -v github.com/terorie/od-database-crawler
+WORKDIR /go/src/github.com/terorie/od-database-crawler
+RUN	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /go/od-database-crawler .
 
-FROM alpine
-WORKDIR /app
-COPY --from=builder /src/binary /app/
-ENTRYPOINT [ "/app/binary", "server" ]
+FROM scratch
+COPY script/ca-certificates.crt /etc/ssl/certs/
```

Comment:
Maybe we can copy the file from alpine instead? Imo, it bloats the repo a bit.

Reply:
scratch has nothing in, no shell, no certs no anything. adding alpine would inflate the image to over 20MB. the program is 14MB. i can tweak it to pull from a remote site but it would have to be static

Reply:
No need to switch from `scratch` to `alpine`. We can use `COPY --from` because there is an `alpine` container available from the previous step.

Reply:
Something like `COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/`

Reply:
Have tweaked this and will make another pull to push the change